packages/eslint/src/generators/init/init.ts

```diff
@@ -44,9 +44,9 @@ function addTargetDefaults(tree: Tree) {
   const nxJson = readNxJson(tree);
 
   nxJson.targetDefaults ??= {};
-  nxJson.targetDefaults.lint ??= {};
-  nxJson.targetDefaults.lint.cache ??= true;
-  nxJson.targetDefaults.lint.inputs ??= [
+  nxJson.targetDefaults['@nx/eslint:lint'] ??= {};
+  nxJson.targetDefaults['@nx/eslint:lint'].cache ??= true;
+  nxJson.targetDefaults['@nx/eslint:lint'].inputs ??= [
```

Comment:
do we want inputs there, or did we decide to not include inputs at all?

Reply:
We still need them when not using PCv3